tick/stateful/node_evaluator.go

```diff
@@ -34,7 +34,9 @@ type NodeEvaluator interface {
 	EvalRegex(scope *Scope, executionState ExecutionState) (*regexp.Regexp, error)
 	EvalTime(scope *Scope, executionState ExecutionState) (time.Time, error)
 	EvalDuration(scope *Scope, executionState ExecutionState) (time.Duration, error)
+	EvalMissing(scope *Scope, executionState ExecutionState) (*ast.Missing, error)
 
+	// TODO: Remove executionState from Type functions type signature
```

Comment:
The reason for this TODO is that I naively tried and it broke a bunch of things. So I set it off temporarily.